core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -426,6 +426,9 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
     padding-right: 10px;
     padding-top: 7px;
   }
+  .rowTwo {
```

Comment:
Can we change it to if possible?
```suggestion
  .row-two {
```

Reply:
Yep, I've renamed the classes 'rowOne' and 'rowTwo' to 'row-one' and 'row-two' respectively.

@Rijuta-s PTAL

Reply:
Thanks